pyproject.toml

```diff
@@ -5,3 +5,14 @@ requires = ["Cython>=0.29.32", "numpy>=1.9.2", "setuptools", "wheel"]
 filterwarnings = [
     "ignore::skbio.util.SkbioWarning",
 ]
+
+[tool.check-manifest]
+ignore = [
+    ".coveragerc",
+    ".dockerignore",
+    ".editorconfig",
+    "RELEASE.md",
+    "REVIEWING.md",
+    "aarch64.Dockerfile",
+    "checklist.py",
```

Comment:
I think `PULL_REQUEST_TEMPLATE.md` should also be in this list.

Reply:
That's already ignored by being in .github (which is a default exclusion for check-manifest)

Reply:
Got it, all good then.